### ISSUE TITLE
docs/add-custom-webpack-config.md error

### ISSUE THREAD

**Issue by benjaminabel** [gatsbyjs/gatsby#6012]
When trying to modify webpack config as described in this doc page, I get the following error:

```
error UNHANDLED EXCEPTION
[1]
[1]
[1]   TypeError: plugins.extractText.extract is not a function
[1]
[1]   - gatsby-node.js:88 Object.exports.onCreateWebpackConfig
[1]     .../gatsby-node.js:88:36
```

**Comment by m-allanson** [comment]
Which version of Gatsby are you using? Can you link to a GitHub repo that shows the error you're seeing?

**Comment by benjaminabel** [comment]
I'm using gatsby v2(next), I'm trying to upgrade my example site for the plugin `@nteract/gatsby-transformer-ipynb`. I need to load `katex.css` however, in v2 I need to configure webpack to do it. Here's the link to the example repo: https://github.com/gatsby-contrib/gatsby-transformer-ipynb/tree/gatsby-v2

**Comment by benjaminabel** [comment]
Yhanks @m-allanson, it seems that you fixed this in #5982.
